### DISCUSSION TITLE
Editor's "Change difficulty" menu item not disabled when no other difficulties are present

### DISCUSSION THREAD

**Discussion by frenzibyte** [ppy/osu#16768] [Q&A] [answered]
As I don't have the time to resolve this yet, I'm opening a discussion thread for the time being.

As the title states, it looks weird for the `Change difficulty` menu item to be enabled/interactive when there are no other beatmaps to switch to, only displaying itself:

![CleanShot 2022-02-04 at 02 43 06](https://user-images.githubusercontent.com/22781491/152447800-0844cd64-b429-421e-ab1c-1a7164dd4c08.png)

Would make sense to be disabled in that case.

**Comment by peppy** [comment, marked as answer]
i've never seen an app do this

**Reply by frenzibyte** [reply]
I don't really have a strong opinion on this so can leave it as-is if it's agreed that's better UX.

**Reply by bdach** [reply]
i would have agreed with disabling when there is zero items in the submenu, but that is not the case.
